src/main/scala/it/pagopa/interop/backendforfrontend/service/types/CatalogProcessServiceTypes.scala

```diff
@@ -184,5 +183,14 @@ object CatalogProcessServiceTypes {
           )
         )
   }
-
+  implicit class EServiceDescriptorSeedConverter(private val seed: EServiceDescriptorSeed) extends AnyVal {
```

Comment:
please do not move code if not necessary 🙏

Reply:
+1